.pre-commit-config.yaml

```diff
@@ -38,7 +38,7 @@ repos:
       - id: markdownlint
         args: ["--config", "mojo/stdlib/scripts/.markdownlint.yaml", "--fix"]
   - repo: https://github.com/keith/pre-commit-buildifier
-    rev: 8.0.3
+    rev: 8.2.0
```

Comment:
FYI this is changed internally already

Reply:
i have to also change it here or CI will be red